core/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -367,6 +382,9 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
         if (reason != null) {
             builder.field(REASON, reason);
         }
+        if (includeGlobalState != null) {
+            builder.field(INCLUDE_GLOBAL_STATE, includeGlobalState);
+        }
```

Comment:
nit: can you move this up before the `state` field and after the `indices` field please?

Reply:
Also, I think that the include_global_state information is important enough to figure in the output when verbose parameter is set to `false`, what do you think?

That mean to update the `if` condition here and to add a new test in `rest-api-spec/test/snapshot.get/10_basic.yml`

Reply:
Really thanks for reviewing this! 
Your mean something like:
```
        final boolean verbose = params.paramAsBoolean("verbose", GetSnapshotsRequest.DEFAULT_VERBOSE_MODE);
        if (verbose || includeGlobalState != null) {
            builder.field(INCLUDE_GLOBAL_STATE, includeGlobalState);
        }
```?
When `verbose` is set to `true` I think, then  `includeGlobalState` will be outputted even though it's `null`. 

If that, I need to add a new test in `rest-api-spec/test/snapshot.status/10_basic.yml` for verbose is `true` and includeGlobalState is `null` I think? But I'm not sure how to make `includeGlobalState` to be null in test. Could you help? Thanks in advance. 👍

Reply:
@liketic Sorry, my asking is not possible since the include_global_state is not stored in the repository medata, so it won't be possible to add the field when the verbose parameter is false.

You can let ignore my second comment, sorry for the noise.

Reply:
Thanks @tlrx . I pushed 244868ab3edf75f0f3c2cc9f63373fe0883135fe